src/arcsf/models/model.py

```diff
@@ -0,0 +1,46 @@
+import torch
+from peft import LoraConfig, PeftModel, get_peft_model
+from transformers import (
+    AutoModelForCausalLM,
+    AutoTokenizer,
+    PreTrainedModel,
+    PreTrainedTokenizer,
+)
+
+
+def load_model_and_tokenizer(
+    model_id: str,
+    peft_kwargs: dict | None = None,
+    **model_kwargs,
+) -> tuple[PreTrainedModel | PeftModel, PreTrainedTokenizer]:
+
+    # Initialise
+    add_token_to_model = True
```

Comment:
Safer for this to start False?

```suggestion
    add_token_to_model = False
```

Reply:
Yes, should be `False` by default (my mistake!)

Reply:
```suggestion
    add_token_to_model = False
```